packages/dashboard/src/components/contextMenu/index.test.tsx

```diff
@@ -0,0 +1,378 @@
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { DndProvider } from 'react-dnd';
+import { TouchBackend } from 'react-dnd-touch-backend';
+
+import { act } from 'react-dom/test-utils';
+import { screen } from '@testing-library/dom';
+import { render } from '@testing-library/react';
+import UserEvent from '@testing-library/user-event';
+
+import ContextMenu, { ContextMenuProps } from './index';
+import { DefaultDashboardMessages } from '~/messages';
+import { DASHBOARD_CONTAINER_ID } from '../grid/getDashboardPosition';
+import { configureDashboardStore } from '~/store';
+
+const renderContextMenu = (args: ContextMenuProps, open?: boolean) => {
+  const ret: { container: HTMLElement | null } = {
+    container: null,
+  };
+
+  act(() => {
+    const { container } = render(
+      <Provider store={configureDashboardStore()}>
```

Comment:
Why do we need to wrap this in the store provider and dnd provider?

Reply:
You're right, we don't, was copied in from another test.